lava-job-runner.py

```diff
@@ -1,4 +1,22 @@
 #!/usr/bin/python
+#
+# Copyright (C) 2014, 2015, 2016 Linaro Limited
```

Comment:
OK, it's not hard to add a list of authors as the history is not very long.  I'd rather add it and update this patch than start chasing everyone until they've added a couple of lines each...